src/unix/linux_like/mod.rs

```diff
@@ -1617,6 +1622,10 @@ cfg_if! {
         pub const STATX_ALL: c_uint = 0x0fff;
         pub const STATX_MNT_ID: c_uint = 0x1000;
         pub const STATX_DIOALIGN: c_uint = 0x2000;
+        pub const STATX_MNT_ID_UNIQUE: c_uint = 0x4000;
+        pub const STATX_SUBVOL: c_uint = 0x8000;
```

Comment:
Small nit, do you mind adding the leading `0x0000` here to keep these in sync with the upstream source?

Reply:
LGTM with that and a squash